app.json

```diff
@@ -27,7 +27,7 @@
   "formation": {
     "web": {
       "quantity": 1,
-      "size": "free"
+      "size": "eco"
```

Comment:
The free tier is no longer an option and caused errors on setup - https://www.heroku.com/pricing